### PR TITLE
Suggested routes inconsistencies

### PR DESCRIPTION
**Pull request by NayanaKamtekar**
I have changed suggested_routes table foreign key.  As mentioned in the task routers, controllers and others should be also called suggested_routes.  But you already suggested changes to ABOD in his PR. I hope he will do that.

### CODE REVIEW

File: db/Schema.dbml [hunk 1]
```diff
@@ -135,8 +136,8 @@ table waypoints {
   sequence_id int     [note: "counts from 0 and up every time a new set of waypoints is added to a route."]
   lon float [ not null ]
   lat float [ not null ]
-  created_on timestamp [not null, default: `now()`]
-  updated_on timestamp [not null, default: `now()`, note: 'TODO: Add ON UPDATE ON UPDATE CURRENT_TIMESTAMP']
+  created_at timestamp [not null, default: `now()`]
+  updated_at timestamp [not null, default: `now()`, note: 'TODO: Add ON UPDATE ON UPDATE CURRENT_TIMESTAMP']
```
Comment (zkwsk) [inline db/Schema.dbml:140]: Really nice that you are finally changing the `created_on` to `created_at`. Remember you need to update the db diagram as well. Refer to the instructions in the readme on how to.

File: src/server/migrations/20190918222221_suggested_routes_foreign_key.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+exports.down = function(knex) {
+  return knex.schema.table("suggested_routes", table => {
+    table.dropForeign("voyage_id");
+    table
+      .foreign("voyage_id", "routes_voyage_id_foreign")
+      .references("voyages.id")
+      .onDelete("CASCADE")
+      .onUpdate("RESTRICT");
+  });
+};
```
Comment (zkwsk) [inline src/server/migrations/20190918222221_suggested_routes_foreign_key.js:23]: Can you please explain this migration? I don't see it perform any changes to my schema for `suggested_routes`?

**Review by zkwsk — CHANGES_REQUESTED** [review]

**Comment by zkwsk** [comment]
Also. Could you please clone the repo directly from the main fork and commit from there from now on? :)

File: src/server/migrations/20190918222221_suggested_routes_foreign_key.js (hunk 2 above)
Comment (NayanaKamtekar) [reply src/server/migrations/20190918222221_suggested_routes_foreign_key.js:23]: It just changes foreign key name from routes_voyage_id_foreign to suggested_routes_voyage_id_foreign

File: db/Schema.dbml (hunk 1 above)
Comment (NayanaKamtekar) [reply db/Schema.dbml:140]: I have modified db diagram as well.

**Review by bytewiz — DISMISSED** [review]
Looks good to me 👍

**Review by zkwsk — APPROVED** [review]

**Comment by zkwsk** [comment]
Merged this based on the fact that @bytewiz had approved it previously :)

**Pull request merged**